### PR TITLE
Disabled pylint

### PR DESCRIPTION
**Pull request by Allexander203**

### CODE REVIEW

File: kiwi_lint/forms.py [hunk 1]
```diff
@@ -1,58 +1,52 @@
...
+from pylint import interfaces
 from pylint_django.utils import node_is_subclass
 
 
 class FormFieldChecker(checkers.BaseChecker):
-    __implements__ = (interfaces.IAstroidChecker,)
+    __implements__ = (interfaces.IAstroidChecker, )
 
-    name = "form-field-label-checker"
+    name = 'form-field-label-checker'
```
Comment (atodorov) [inline kiwi_lint/forms.py:9]: These changes are unrelated. Need to revert them and rebase to latest master !

File: tcms/testcases/models.py [hunk 2]
```diff
@@ -339,14 +339,14 @@ def add_cc(self, email_addrs):
...
-            return []
+            return [] #pylint:disable=Protected-access
         return self.cc_list.split(MultipleEmailField.delimiter)
 
     def remove_cc(self, email_addrs):
         """Remove one or more email addresses from EmailSettings' CC list
 
         If any email_addr is unknown, remove_cc will keep quiet.
-
+#pylint:disable=Protected-access
```
Comment (atodorov) [inline tcms/testcases/models.py:349]: This is a comment inside a documentation string for the function `remove_cc`. It doesn't do anything. Remove it.

File: tcms/testcases/models.py [hunk 3]
```diff
@@ -339,14 +339,14 @@ def add_cc(self, email_addrs):
     def get_cc_list(self):
         """ Return the whole CC list """
         if not self.cc_list:
-            return []
+            return [] #pylint:disable=Protected-access
```
Comment (atodorov) [inline tcms/testcases/models.py:342]: Dos this have anything to do with silencing the `form-field-help-text-used` ? It is not even in the right location.

File: tcms/testcases/models.py [hunk 4]
```diff
@@ -355,7 +355,7 @@ def remove_cc(self, email_addrs):
             email_addrs = [email_addrs]
         for address in email_addrs:
             if address in emailaddr_list:
-                emailaddr_list.remove(address)
+                emailaddr_list.remove(address)#pylint:disable=Protected-access
```
Comment (atodorov) [inline tcms/testcases/models.py:358]: Dos this have anything to do with silencing the `form-field-help-text-used` ? It is not even in the right location.

**Review by atodorov — CHANGES_REQUESTED** [review]
Multiple changes requested. Also:

- squash all commits together so that there is only 1 left at the end
- update commit log message to be more meaningful
- update PR title to something more meaningful

File: tcms/testcases/models.py [hunk 5]
```diff
@@ -356,6 +356,5 @@ def remove_cc(self, email_addrs):
         for address in email_addrs:
             if address in emailaddr_list:
                 emailaddr_list.remove(address)
-
```
Comment (atodorov) [inline tcms/testcases/models.py:359]: Unrelated change, restore it like it was before.

File: tcms/testcases/models.py [hunk 6]
```diff
@@ -279,7 +279,7 @@ class BugSystem(TCMSActionModel):
 """,
     )
 
-    api_url = models.CharField(
+    api_url = models.CharField(# pylint:disable=form-field-help-text-used
```
Comment (atodorov) [inline tcms/testcases/models.py:282]: All of the comments trigger flake8 errors. Check the logs or see Makefile how to execute flake8 locally.

**Review by atodorov — CHANGES_REQUESTED** [review]

File: tcms/testcases/models.py [hunk 7]
```diff
@@ -355,7 +355,7 @@ def remove_cc(self, email_addrs):
             email_addrs = [email_addrs]
         for address in email_addrs:
             if address in emailaddr_list:
-                emailaddr_list.remove(address)
+                emailaddr_list.remove(address)  # pylint:disable=Protected-access
```
Comment (asankov) [inline tcms/testcases/models.py:358]: this is not needed. there is no alert to silence on this line. revert it.

File: tcms/testcases/models.py [hunk 8]
```diff
@@ -339,7 +339,7 @@ def add_cc(self, email_addrs):
     def get_cc_list(self):
         """ Return the whole CC list """
         if not self.cc_list:
-            return []
+            return []  # pylint:disable=Protected-access
```
Comment (asankov) [inline tcms/testcases/models.py:342]: this is not needed. there is no alert to silence on this line. revert it.

File: kiwi_lint/forms.py [hunk 9]
```diff
@@ -1,4 +1,5 @@
 from pylint import checkers, interfaces
+
```
Comment (asankov) [inline kiwi_lint/forms.py:2]: revert this

**Review by asankov — CHANGES_REQUESTED** [review]

**Review by asankov — DISMISSED** [review]

**Review by atodorov — CHANGES_REQUESTED** [review]
Changes look good however the commit log still doesn't tell us much about why or what are you doing. See from previous review item:

- update commit log message to be more meaningful

"Successfully disabled pylint" doesn't mean anything for anyone else but you.

**Review by atodorov — APPROVED** [review]

**Pull request merged**